OrbitVulkanLayer/EntryPoints.cpp

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "DeviceManager.h"
+#include "DispatchTable.h"
+#include "QueueManager.h"
+#include "SubmissionTracker.h"
+#include "TimerQueryPool.h"
+#include "VulkanLayerController.h"
+#include "absl/base/casts.h"
+#include "vulkan/vk_layer.h"
+#include "vulkan/vulkan.h"
+
+/*
+ * The big picture:
+ * This is the main entry point for Orbit's Vulkan layer. The layer is structured as follows:
+ * ° All instrumented Vulkan functions will hook into implementations found here
```

Comment:
Use "-" instead. (This is the "degree" character, right? That shouldn't be used as a bullet point.)

Reply:
Done. Thanks